package.json

```diff
@@ -57,13 +57,14 @@
     "@sveltejs/adapter-auto": "^3.0.0",
     "@sveltejs/adapter-static": "^3.0.5",
     "@sveltejs/kit": "^2.0.0",
-    "@sveltejs/vite-plugin-svelte": "^4.0.0-next.7",
+    "@sveltejs/vite-plugin-svelte": "^4.0.0",
     "@tauri-apps/cli": "2.0.0-rc.16",
     "@testing-library/webdriverio": "^3.2.1",
     "@types/eslint": "^9.6.0",
     "@types/eslint-config-prettier": "^6.11.3",
     "@types/eslint__js": "^8.42.3",
     "@types/onscan.js": "^1.5.6",
+    "@types/sql.js": "^1.4.9",
```

Comment:
`sql.js` is a browser-based `sqlite` we will be using to support unit testing w/o the backend sqlite DB / Rust. This is the typescript types package for it. The actual plugin is included below.